#! /bin/sh /usr/share/dpatch/dpatch-run
## logs.dpatch.dpatch by  <jetty@packaging-u1040-32.dev.opengeo.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: removing opengeologfile appender

@DPATCH@

diff -rc opengeo-suite-data-orig/data_dir/logs/DEFAULT_LOGGING.properties opengeo-suite-data/data_dir/logs/DEFAULT_LOGGING.properties
*** opengeo-suite-data-orig/data_dir/logs/DEFAULT_LOGGING.properties	2011-04-08 21:06:32.000000000 +0000
--- opengeo-suite-data/data_dir/logs/DEFAULT_LOGGING.properties	2011-04-08 21:07:11.000000000 +0000
***************
*** 1,7 ****
  ## This log4j configuration file needs to stay here, and is used as the default logging setup
  ## during data_dir upgrades and in case the chosen logging config isn't available.
  
! log4j.rootLogger=WARN, geoserverlogfile, opengeologfile, stdout
  
  log4j.appender.stdout=org.apache.log4j.ConsoleAppender
  log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
--- 1,7 ----
  ## This log4j configuration file needs to stay here, and is used as the default logging setup
  ## during data_dir upgrades and in case the chosen logging config isn't available.
  
! log4j.rootLogger=WARN, geoserverlogfile, stdout
  
  log4j.appender.stdout=org.apache.log4j.ConsoleAppender
  log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
diff -rc opengeo-suite-data-orig/data_dir/logs/GEOSERVER_DEVELOPER_LOGGING.properties opengeo-suite-data/data_dir/logs/GEOSERVER_DEVELOPER_LOGGING.properties
*** opengeo-suite-data-orig/data_dir/logs/GEOSERVER_DEVELOPER_LOGGING.properties	2011-04-08 21:06:32.000000000 +0000
--- opengeo-suite-data/data_dir/logs/GEOSERVER_DEVELOPER_LOGGING.properties	2011-04-08 21:06:52.000000000 +0000
***************
*** 15,21 ****
  ## FATAL                 SEVERE
  ## OFF                   OFF
  
! log4j.rootLogger=WARN, geoserverlogfile, opengeologfile, stdout
  
  log4j.appender.stdout=org.apache.log4j.ConsoleAppender
  log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
--- 15,21 ----
  ## FATAL                 SEVERE
  ## OFF                   OFF
  
! log4j.rootLogger=WARN, geoserverlogfile, stdout
  
  log4j.appender.stdout=org.apache.log4j.ConsoleAppender
  log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
diff -rc opengeo-suite-data-orig/data_dir/logs/GEOTOOLS_DEVELOPER_LOGGING.properties opengeo-suite-data/data_dir/logs/GEOTOOLS_DEVELOPER_LOGGING.properties
*** opengeo-suite-data-orig/data_dir/logs/GEOTOOLS_DEVELOPER_LOGGING.properties	2011-04-08 21:06:32.000000000 +0000
--- opengeo-suite-data/data_dir/logs/GEOTOOLS_DEVELOPER_LOGGING.properties	2011-04-08 21:07:03.000000000 +0000
***************
*** 15,21 ****
  ## FATAL                 SEVERE
  ## OFF                   OFF
  
! log4j.rootLogger=WARN, geoserverlogfile, opengeologfile, stdout
  
  log4j.appender.stdout=org.apache.log4j.ConsoleAppender
  log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
--- 15,21 ----
  ## FATAL                 SEVERE
  ## OFF                   OFF
  
! log4j.rootLogger=WARN, geoserverlogfile, stdout
  
  log4j.appender.stdout=org.apache.log4j.ConsoleAppender
  log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
diff -rc opengeo-suite-data-orig/data_dir/logs/PRODUCTION_LOGGING.properties opengeo-suite-data/data_dir/logs/PRODUCTION_LOGGING.properties
*** opengeo-suite-data-orig/data_dir/logs/PRODUCTION_LOGGING.properties	2011-04-08 21:06:32.000000000 +0000
--- opengeo-suite-data/data_dir/logs/PRODUCTION_LOGGING.properties	2011-04-08 21:06:58.000000000 +0000
***************
*** 15,21 ****
  ## FATAL                 SEVERE
  ## OFF                   OFF
  
! log4j.rootLogger=WARN, geoserverlogfile, opengeologfile, stdout
  
  log4j.appender.stdout=org.apache.log4j.ConsoleAppender
  log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
--- 15,21 ----
  ## FATAL                 SEVERE
  ## OFF                   OFF
  
! log4j.rootLogger=WARN, geoserverlogfile, stdout
  
  log4j.appender.stdout=org.apache.log4j.ConsoleAppender
  log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
diff -rc opengeo-suite-data-orig/data_dir/logs/VERBOSE_LOGGING.properties opengeo-suite-data/data_dir/logs/VERBOSE_LOGGING.properties
*** opengeo-suite-data-orig/data_dir/logs/VERBOSE_LOGGING.properties	2011-04-08 21:06:32.000000000 +0000
--- opengeo-suite-data/data_dir/logs/VERBOSE_LOGGING.properties	2011-04-08 21:07:07.000000000 +0000
***************
*** 15,21 ****
  ## FATAL                 SEVERE
  ## OFF                   OFF
  
! log4j.rootLogger=WARN, geoserverlogfile, opengeologfile, stdout
  
  log4j.appender.stdout=org.apache.log4j.ConsoleAppender
  log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
--- 15,21 ----
  ## FATAL                 SEVERE
  ## OFF                   OFF
  
! log4j.rootLogger=WARN, geoserverlogfile, stdout
  
  log4j.appender.stdout=org.apache.log4j.ConsoleAppender
  log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
